SHA256SUMS

```diff
@@ -8,3 +8,22 @@ b938f277c97dfa9e9106a9cf536ba3f6bbfebddebcdcd541fadb2d2e30f95c12  ios-libevent-2
 c20d5b71624681e7eecc355d95fdcbf128f5ae7c71a2575788fb770dd067b9dc  ios-curl-7.63.0+2.tar.gz
 973531e391c02f6699ff1232826367e9e3e5d05eb4f28dcf37f1a97102cf38d0  android-measurement-kit-0.9.1+1.tar.gz
 e20ba3cc92442606a221311993ed720f00615a2ba5c7bb63adf42081b446aa7a  ios-measurement-kit-0.9.1+1.tar.gz
+4e0ecf66ff3fdaa7cd6d903fc402a0e54c4e2c0c0f64677d80f72622a8cc9a1b  linux-geoip-api-c-1.6.12-4.tar.gz
+8430f6e9a77c7195b1147dee4984221c3484c1660ea86088e930610cd6c97f5d  macos-curl-7.63.0+2.tar.gz
+e1b63f8e5a747c439e63974fe875799baf4bdfaa8fc5e61777f47ca18d749242  macos-geoip-api-c-1.6.12-3.tar.gz
+550d3f6ba10507974b5af55983a6e5e4203fc7ba26f6fca65081bf9721804048  macos-libevent-2.1.8+4.tar.gz
+1f397334ee7d2cb142dae8ba79da1f0e8e1ce76d9579fdbfd2b186fab2201961  macos-libevent-2.1.8-4.tar.gz
```

Comment:
What is going on here? Why there are two duplicate packages? (Did you start building using as base the current master?)

Reply:
Yeah sorry about that. I had a dirty MK_DIST and MK_BUILD dirs and didn't initially notice that `./package` script created packages of those.

Reply:
I didn't realise either that the problem was so simple. (Jet lag doesn't help.) All good :-)